src/setup-bot.ts

```diff
@@ -73,6 +78,29 @@ export async function setupBot (): Promise<void> {
     }),
   )
 
+  if (process.env['CHATOPERA_CLIENTID'] && process.env['CHATOPERA_SECRET']) {
+    const chatoperaFiltersGroup = process.env['CHATOPERA_FILTERS_GROUP']?.split(',')
```

Comment:
I think we need to make the filters to be configurable for each project?

If so, then it seems that it will be a good idea to put those configs in the below settings:

https://github.com/kaiyuanshe/osschat/blob/7a7d58fa5a6d659e97a4436bd8d85e786ede9085/src/config-projects/tencent.ts#L11-L13

Reply:
Ok, get your ideas, for each organization in `config-projects`, they are **users**, the bot should be configured per user, each user could has just one BOT to handle the chats.
 
https://github.com/kaiyuanshe/osschat/tree/7a7d58fa5a6d659e97a4436bd8d85e786ede9085/src/config-projects

Reply:
For each `config-projects`, the organization，e.g. apache, can provide two environment variables:

``` 
WCP_{GITHUB_OWNER}_CLIENT_ID=xxx
WCP_{GITHUB_OWNER}_SECRET=xxx
```

For example

```
WCP_APACHE_CLIENT_ID=xxx
WCP_APACHE_SECRET=xxx
```

Then, OSSChat instance would detect the config for projects which starts with `apache/` in GitHub.

[GitHub Repo, Wechat Groups] => Environment variables => OSSChat => Chatopera BOT

Reply:
Yes, that's the idea.

And I'd like to suggest that we can share one Chatopera bot backend for all projects?

If we can, then all projects can focus on the FAQ YAML data, and our OSSChat can take care of different project for the users